### DISCUSSION TITLE
active learning framework in doccano

### DISCUSSION THREAD

**Discussion by KINJALMARU16** [doccano/doccano#1757] [Q&A]
Hello,
How can we create active learning framework  in doccano?

**Comment by Hironsan** [comment]
doccano supports REST API. So it would be useful to implement active learning.

![image](https://user-images.githubusercontent.com/6737785/160765634-b82148b2-f2d0-494c-bf7f-ddbe8051a2c4.png)

The `Label` model has a `prob` field. It would be useful.

https://github.com/doccano/doccano/blob/350bc5e95fa459d1dc124a320cb8e34385b73021/backend/labels/models.py#L16-L26

It would have been better to have a `confidence` field in the `Example` model.
